sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -846,6 +846,30 @@ class SessionCatalog(
     case None => fromCatalogTable(viewInfo.tableMeta, isTempView = true)
   }
 
+  private def buildViewDDL(metadata: CatalogTable, isTempView: Boolean): String = {
+    val isGlobalTemp = metadata.identifier.database.exists(_ == globalTempViewManager.database)
+    val viewType = if (isTempView && isGlobalTemp) {
```

Comment:
I think this only applies to the permanent view? We don't store temp view in the cataog.

Reply:
Ah, sounds good. this will make the code simpler.